plugins/opensslcryptoplugin/cryptoplugin_common.cpp

```diff
@@ -101,6 +108,17 @@ namespace {
         }
         return 0; // no cipher sessions available.
     }
+
+    static const EVP_MD *getEvpDigestFunction(Sailfish::Crypto::CryptoManager::DigestFunction digestFunction) {
```

Comment:
in anonymous namespace so doesn't need static qualifier I guess

Reply:
No, it indeed doesn't.